cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesSubset.java

```diff
@@ -71,11 +72,26 @@ public String getIdentifier() {
         return identifier;
     }
 
+    public String getProfile() {
+        return profileName;
+    }
+
     public boolean isValidName(String contextName) {
         return contextName.contains(validName0) || contextName.contains(validName1);
     }
 
     private static boolean isBoundary(String contextName) {
         return contextName.contains("_BD") || contextName.contains("BOUNDARY");
     }
+
+    private final String identifier;
+    private final String validName0;
+    private final String validName1;
+    private String profileName;
+    private final Map<String, String> profile = ImmutableMap.of(
```

Comment:
I think you should make this map static. Change also the case of `profile` to upper case.

Reply:
profile is used in the constructor of the enum

Reply:
Done